index.html

```diff
@@ -27,15 +42,18 @@
 <body>
   <section id="abstract">
     <p>
-      This report documents recommendations for ActivityPub and Activity Streams 2.0 to improve Trust and Safety on the Fediverse.
+      This report documents recommendations for ActivityPub and Activity Streams 2.0 to improve Trust and Safety on the
+      Fediverse.
     </p>
   </section>
   <section id="sotd">
     <p>This is required.</p>
```

Comment:
This part is a little confusing to me.  Resolve/close the comment if i'm misunderstanding but it looks like a placeholder? Would the boilerplate still show up if you removed this specific <p> line?